support.js

```diff
@@ -7,6 +7,17 @@
 if (Cypress.env('coverage') === false) {
   console.log('Skipping code coverage hooks')
 } else {
+  const sendCoverageObject = (coverage) => {
+    if (!coverage) {
+      return
+    }
+    cy.task('combineCoverage', coverage).then(response => {
+      cy.log(`**${response.covered.s}**/**${response.total.s}** statements`)
```

Comment:
A test that this `cy.log` works as expected with the correct output would be nice to have, like in `submit_spec`: https://github.com/cypress-io/cypress/blob/876fe5bbd890164e2bc901f5376cb69d0aaad583/packages/driver/test/cypress/integration/commands/actions/submit_spec.coffee#L168-L172

Reply:
good observation about logs test, I might start create test subfolders with test projects. It will be hard to test everything since most of the stuff happens after each or after all tests